packages/ui/src/Box.tsx

```diff
@@ -274,4 +275,8 @@ export class Box extends React.Component<BoxProps, {}> {
     }
     return box;
   }
+
+  static defaultProps = {
+    zIndex: "auto",
```

Comment:
This would be a surprising change and probably will wind up with some random, unexpected, and hard to track down effects. I like having the zIndex field, but not defaulting it to auto.

Reply:
Agreed. And moving it inline like the other defaults.

Reply:
Yea I understand those concerns and not really sure how to avoid it. "auto" is the default z-index value for regular html elements as opposed to the zindex of 1 that View is instantiated with. The double edge sword I've seen is View is essentially react-native's div which means we build everything with it but it also doesn't follow the normal z-index rules because it's not inheriting any z-values from the parent element. Definitely open to ideas.

I'll also continue messing with the default assignment being inline. I haven't had any luck but I'm sure there's a way.

Reply:
Can you not pass in the zIndex prop to the Boxes surrounding the calendar and get the effect you're looking for? I haven't dug into this very much.

Reply:
So those are the rules but the key is the very last section. Unfortunately, the calendar is positioned as absolute so it throws a wrench into this. The z-index: auto creates a new local stacking context but I think the better option is to just set the z-index to auto for the box that surrounds the calendar library and then again for the box that surrounds our ferns-ui calendar in the flourish app.